apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -75,14 +81,17 @@ export function toWorkflowsMinifiedDtos(templates: NotificationTemplateEntity[])
 }
 
 function toStepResponseDto(step: NotificationStepEntity): StepResponseDto {
+  const stepName = step.name || 'Missing Name';
+
   return {
-    name: step.name || 'Missing Name',
-    stepUuid: step._templateId,
+    _id: step._templateId,
+    slug: `${slugifyName(stepName)}_${encodeBase62(step._templateId)}`,
```

Comment:
same here for step, 
add prefix

same here for step
remove slugifyName(stepName), slug should be `{prefix}{shortId}` in order to support future refactoring, the client should structure the actual slug with {name}{slug}